make html Fails When Making Pages

I've just started using pelican (and this theme). Upon adding a page (eg. content/pages/about.md), `make html` produces the following error:

```
CRITICAL: UndefinedError: 'article' is undefined
Makefile:65: recipe for target 'html' failed
make: *** [html] Error 1
```

It seems that this is caused by lines 24 and 25 of `templates/pages.html`. Those lines were introduced recently as part of PR #27.

I made the change in the patch below in my local copy (changing `article` to`page`) and now `make html` succeeds. However, since I'm new to both pelican and attila, I'm not completely sure that this change is correct and doesn't break something else.

```
--- a/attila-1.3/templates/page.html
+++ b/attila-1.3/templates/page.html
@@ -21,8 +21,8 @@
   {% else %}
     {% set selected_cover = SITEURL+"/"+HEADER_COVER %}
   {% endif %}
-{% elif article.color %}
-  {% set selected_color = article.color %}
+{% elif page.color %}
+  {% set selected_color = page.color %}
 {% elif HEADER_COLOR %}
   {% set selected_color = HEADER_COLOR %}
 {% endif %}
```

--- a/templates/page.html
+++ b/templates/page.html
@@ -21,8 +21,8 @@
   {% else %}
     {% set selected_cover = SITEURL+"/"+HEADER_COVER %}
   {% endif %}
-{% elif article.color %}
-  {% set selected_color = article.color %}
+{% elif page.color %}
+  {% set selected_color = page.color %}
 {% elif HEADER_COLOR %}
   {% set selected_color = HEADER_COLOR %}
 {% endif %}

